src/main/features/claim/routes/pay.ts

```diff
@@ -93,10 +95,10 @@ async function successHandler (res, next) {
       logger.error(`missing consent not given role for user, User Id : ${user.id}`)
     }
 
-    let features: string
-    if (await featureTogglesClient.isFeatureToggleEnabled(user, roles, 'cmc_admissions')) {
+    let features: string = ''
+    launchDarklyClient.callFeatureFlag(user, config.get<string>('launchDarkly.featureFlag-keys.admissions'), function () {
```

Comment:
this config.get is too long either extract to a variable or make it a constant

also do you need an anonymous function or can you use an arrow function?
```suggestion
    launchDarklyClient.callFeatureFlag(user, config.get<string>('launchDarkly.featureFlag-keys.admissions'), () => {
```